react/details/package.json

```diff
@@ -1,24 +1,28 @@
 {
-  "name": "conference-details-widget",
+  "name": "all-typed-blob-entity-details-widget",
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@entando/apimanager": "^4.1.0",
     "@material-ui/core": "^4.4.2",
     "@material-ui/icons": "^4.4.3",
     "@material-ui/styles": "^4.4.3",
     "clsx": "^1.0.4",
     "i18next": "^17.0.15",
+    "keycloak-js": "^7.0.1",
     "prop-types": "^15.7.2",
     "react": "^16.9.0",
     "react-dom": "^16.9.0",
     "react-i18next": "^10.12.5",
+    "react-keycloak": "^7.0.1-191016",
     "react-scripts": "3.1.1"
   },
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test --verbose",
     "eject": "react-scripts eject",
+    "keycloak": "node ./scripts/keycloak.config.js",
```

Comment:
what this script does? where's this file?

Reply:
removed